client/battle/CBattleInterface.cpp

```diff
@@ -1630,6 +1638,9 @@ void CBattleInterface::setHoveredStack(const CStack *stack)
 
 void CBattleInterface::activateStack()
 {
+	if(!battleActionsStarted)

```

Comment:
making some getter method for such things will be nice

Reply:
But... this variable is private

Reply:
yep that makes a thing a bit strange, like before just thought that in the future refactoring this can be helpfull